frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/dialogs/DeleteBranchOnSlideOutSuggestionDialog.kt

```diff
@@ -0,0 +1,82 @@
+package com.virtuslab.gitmachete.frontend.actions.dialogs
+
+import com.intellij.openapi.project.Project
+import com.intellij.openapi.ui.DialogWrapper
+import com.intellij.openapi.ui.Messages
+import com.intellij.ui.layout.panel
+import com.virtuslab.gitmachete.frontend.actions.base.BaseSlideOutBranchAction.DELETE_LOCAL_BRANCH_ON_SLIDE_OUT_GIT_CONFIG_KEY
+import com.virtuslab.gitmachete.frontend.resourcebundles.GitMacheteBundle.format
+import com.virtuslab.gitmachete.frontend.resourcebundles.GitMacheteBundle.getString
+import java.awt.event.KeyEvent
+import javax.swing.Action
+import javax.swing.JTextPane
+
+data class SlideOutOptions(
+    @get:JvmName("shouldRemember")
+    val remember: Boolean = false,
+    @get:JvmName("shouldDelete")
+    val delete: Boolean = false)
+
+class DeleteBranchOnSlideOutSuggestionDialog constructor(project: Project) :
+    DialogWrapper(project, /* canBeParent */ true) {
+
+  private var remember = false
+  private var delete = false
+
+  init {
+    title = getString("action.GitMachete.BaseSlideOutBranchAction.deletion-suggestion-dialog.title")
+    setResizable(false)
+    super.init()
+  }
+
+  fun showAndGetSlideOutOptions() = if (showAndGet()) SlideOutOptions(remember, delete) else null
+
+  override fun createCenterPanel() =
+      panel {
+        row {
+          createNoteOrCommentRow(
+              Messages.configureMessagePaneUi(
+                  JTextPane(),
+                  format(
+                      getString(
+                          "action.GitMachete.BaseSlideOutBranchAction.deletion-suggestion-dialog.note"),
```

Comment:
Pls verify, but I don't think Checker (or Kotlin compiler itself) is able to spot typos in property key in Kotlin files... consider adding a script that would at least verify that all `action.GitMachete....` strings from \*.kt files occur in GitMacheteBundle.properties

Reply:
![image](https://user-images.githubusercontent.com/19799111/103522175-48d7a200-4e7a-11eb-83c3-52cc230e519c.png)

results with:

`java.util.MissingResourceException: Can't find resource for bundle java.util.PropertyResourceBundle, key actio.GitMachete.BaseSlideOutBranchAction.deletion-suggestion-dialog.remember-choice`

---

good idea